schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -657,6 +657,11 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-06-01/Microsoft.Network.json#/resourceDefinitions/networkInterfaces" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-06-01/Microsoft.Network.json#/resourceDefinitions/routeTables" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-06-01/Microsoft.Network.json#/resourceDefinitions/applicationGateways" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-06-01/Microsoft.Network.json#/resourceDefinitions/connections" },
```

Comment:
@bmoore-msft Do you have any perf concerns over the adding of this many resource type / api versions?